app/views/feature_flags/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<table class="govuk-table feature-table">
+  <thead class="govuk-table__head">
+  <tr class="govuk-table__row">
+    <th class="govuk-table__header" scope="col">Name</th>
+    <th class="govuk-table__header" scope="col">Description</th>
+    <th class="govuk-table__header centered" scope="col">Development</th>
+    <th class="govuk-table__header centered" scope="col">Test</th>
+    <th class="govuk-table__header centered" scope="col">Server test</th>
+    <th class="govuk-table__header centered" scope="col">Staging</th>
+    <th class="govuk-table__header centered" scope="col">Production</th>
+  </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+  <% @features.each do |feature| %>
+    <tr class="booking govuk-table__row">
+      <td class="name govuk-table__cell">
+        <%= feature.name %>
+      </td>
+      <td class="date govuk-table__cell">
+        <%= feature.description %>
+      </td>
+      <td class="subject govuk-table__cell environment">
```

Comment:
I think this could be more concise and future-proof by creating some convenience methods in the model:

```
<th class="govuk-table__header" scope="col">Name</th>
<th class="govuk-table__header" scope="col">Description</th>

<% FeatureFlags.environments do |environment| %>
  <th class="govuk-table__header centered" scope="col"><%= environment %></th>
<% end %>

...


<% FeatureFlags.environments do |environment| %>
   <div class="centered <%= feature.enabled_for?(environment) ? "check" : "cross" %>"></div>
<% end %>
```